### DISCUSSION TITLE
Super Collections & Sub Collections

### DISCUSSION THREAD

**Discussion by ma3ain** [payloadcms/payload#2395] [Feature Requests & Ideas]
I would like to request a feature which is ability to have Collections contained _within_ Collections.
When a Collection contains one or more Collections, the parent Collection is the **Super Collection** and the child Collections are **Sub Collections**. 
That will help large projects to be more organized & systematic.

**Comment by IRediTOTO** [comment]
what is the difference  when use relationship field ? :\
